src/js/pieChart.js

```diff
@@ -68,21 +72,41 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
         },
         selectors: {
             pie: ".floec-ca-pieChart-pie",
-            slice: ".floec-ca-pieChart-legend"
+            legend: ".floec-ca-pieChart-legend"
+        },
+        listeners: {
+            "onCreate.renderTemplate": "floe.chartAuthoring.pieChart.renderTemplate"
```

Comment:
It might make more sense to move this event firing to ```floe.chartAuthoring.templateInjection``` component since they are more related? In there, this line can be simplified to:
```
listeners: {
    "onCreate.injectTemplate": ...
    "onCreate.onTemplateInjected": {
    	listener: "{that}.events.onTemplateInjected.fire",
    	priority: "after:injectTemplate"
    }
}
```

Also, what about having a more specific event name such as ```onTemplateRendered``` or ```onTemplateInjected``` instead of ```onParentReady```?